### DISCUSSION TITLE
Status of cyclonedx-conan and cyclonedx-cli

### DISCUSSION THREAD

**Discussion by andreas-hilti** [jkowalleck/jkowalleck#3] [HELLO]
Hi Jan,

Given that you were the last one to contribute to cyclonedx-conan, I'm approaching you, but let me know if I'm addressing the wrong person.
What is the state/plan w.r.t. [cyclonedx-conan](https://github.com/CycloneDX/cyclonedx-conan) and [cyclonedx-cli](https://github.com/CycloneDX/cyclonedx-cli)? Both of them are useful for my workflows, however they don't seem to get much attention/many commits recently.
I have opened several PRs (to fix minor bugs) on both repos, but I never got any feedback. And at least for the cli, the issues start piling up...

From my understanding, at least the cli would be a central piece in the cyclonedx environment, as it is the only tool that allows to combine SBOMs (generated by different tools) and to modify them.

(cyclonddx-conan on the other hand might actually need a bit less maintainance as most of the functionality is offloaded to conan; however, given that also for C++ package managers are used more and more, it would be valuable to support them.)

Thanks in advance for your time.

Regards,
Andreas

**Comment by jkowalleck** [comment]
Hello @andreas-hilti ,
Would you come to the projects' slack? Use <https://cyclonedx.org/slack/invite> to join.
Let's have a chat :-D
